Enable curly bracket spacing for jsx-curly-spacing children

kngroo opened the pull request:
##### *To:*
@cansin 

##### *What:*
Add curly brace spacing for jsx children as per here (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md)

##### *Trello:*
https://trello.com/c/xRAx7WGU/1841-tweak-eslint-to-put-space-inside-curly-for-children

##### *What did you test:*
Ensure unit tests are passing.
From website-django repo, run `yarn run lint` ~1300 errors from new rule.
Run `yarn run lint-fix` and ensure most errors are fixed - `git diff -w origin/master` should not show any diffs for files that were autofixed.

##### *What dashboards will you be monitoring:*
Standard

Commit:
fix: Enable children curly bracket spacing attribute for jsx-curly-spacing rule.

Commit:
Enable children curly bracket spacing attribute for jsx-curly-spacing rule

cansin reviewed (approved):
LGTM

cansin commented:
I tried closing/re-opening to see if CircleCI would catch the PR. I guess it won't until this branch is rebased with the master.

cansin commented:
I don't know why ci/circleci is not running for this branch yet. I suspect it is because it doesn't have the `.circleci/config.yml` file yet. @kngroo can you rebase from master to see if it will unstuck the PR? Thank you!

kngroo committed:
Merge branch 'master' into improvement/add-jsx-curly-spacing-children-rule

Commit:
Merge branch 'improvement/add-jsx-curly-spacing-children-rule' of github.com:udemy/js-tooling into improvement/add-jsx-curly-spacing-children-rule

Commit:
Publish

 - babel-polyfill-udemy-website@0.6.3
 - babel-preset-udemy-website@0.6.3
 - eslint-config-udemy-react-addons@3.1.11
 - eslint-config-udemy-website@4.0.5

kngroo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Tweak ESLint to put space inside curly for children](https://trello.com/c/xRAx7WGU/1841-tweak-eslint-to-put-space-inside-curly-for-children)
